DSCResources/MSFT_xCertificateImport/MSFT_xCertificateImport.schema.mof

```diff
@@ -0,0 +1,8 @@
+[ClassVersion("1.0"), FriendlyName("xCertificateImport")]
+class MSFT_xCertificateImport : OMI_BaseResource
+{
+[Key] string Thumbprint;
+[Key] string Path;
+[write,ValueMap{"LocalMachine"},Values{"LocalMachine"}] string Location;
+[write] string Store;
```

Comment:
So in this case the *Location*, *Store* and *Thumbprint* would be the keys since that is what uniquely identifies the certificate once it is installed on the computer? Most of my implementation was a straight copy of the xPfxImport-resource so I didn't really reflect much on the meaning of the keys. Perhaps the xPfxImport should also be updated?